README.md

```diff
@@ -74,7 +74,6 @@ look like this in the workspace:
     zv925hd6723/
       metadata/
         contentMetadata.xml
-        descMetadata.xml
```

Comment:
as of this PR, there are still a couple spots that try to load the file (`normalize_data.rb` and `load_geoserver.rb`).  but afaik, nothing actually generates it anymore?